api/kessel/inventory/v1beta1/notifications_integration.proto

```diff
@@ -14,7 +14,7 @@ option java_package = "org.project_kessel.api.inventory.v1beta1";
 
 message NotificationsIntegration {
   // Metadata about this resource
-  Metadata metadata = 1 [ (google.api.field_behavior) = REQUIRED, (validate.rules).any.required = true ];
+  Metadata metadata = 1;
```

Comment:
```suggestion
  Metadata metadata = 1 [ (validate.rules).any.required = true ];
```
We will still require the required annotation. It generates code to do the validation.

Reply:
metadata should not be required.  There are no fields that are required for one to pass in.